spec/fake_stripe/stub_app_spec.rb

```diff
@@ -48,6 +48,12 @@
         Stripe::Charge.create
       end.to change(FakeStripe, :charge_count).by(1)
     end
+
+    it 'returns an error for a negative amount' do
+      expect do
+        Stripe::Charge.create(amount: -100)
+      end.to raise_error(Stripe::InvalidRequestError)
+    end
```

Comment:
Do you also want to verify that `FakeStripe.charge_count` doesn't change?

Reply:
That would be nice. Do you know if it is possible to do both a check for `raise_error` and `change` in one assertion? Or do I need to make another separate assertion?

Reply:
I've pushed up 7822972 which adds an assertion for not incrementing the charge counter.